dts/arc/quark_se_c1000_ss.dtsi

```diff
@@ -0,0 +1,65 @@
+#include "skeleton.dtsi"
+
+#define __SIZE_K(x) (x * 1024)
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "snps,arcem4";
+			reg = <1>;
+		};
+
+		core_intc: arcem4-intc@0 {
+			compatible = "snps,arcem4-intc";
```

Comment:
lets rename this to:
compatible = "snps,arcv2-intc";

Reply:
ok